src/app/dataset/dataset.component.ts

```diff
@@ -94,20 +94,22 @@ export class DatasetComponent implements OnInit, OnDestroy {
   setReportMsg(req: SimpleReportRequest): void {
     if (req.message) {
       this.reportMsg = req.message;
-    } else if (req.taskId && req.topology) {
+    }
+    if (req.taskId && req.topology) {
```

Comment:
I am not sure if I understand this correctly, but does this mean that the reportMsg (req.message) can be overwritten with "Report is empty"?

Reply:
Instead of the "else" the report will show if there is either a message or a list of errors